vignettes/programming_strategy.Rmd

```diff
@@ -655,35 +653,49 @@ fun_xxx <- function(dataset, some_param, other_param) {
     other_param = other_param
```

Comment:
I think the line
```
  new_fun_xxx(
```
is missing. Otherwise, it looks strange.

Reply:
I don't think we want that there right?  The example is the original function that sends the message to the user about using the new function

Reply:
I assume we wanted to show that the new function should be called.

Reply:
Maybe we should also mention somewhere in the text that the code of the deprecated function should be replaced with the call to the new function (if possible).

Reply:
Hey @bundfussr, sorry I am again a bit lost with your comment.  If you want to make the change, please implement and I can check it out :)

Reply:
so event() would need to go here?  
![image](https://github.com/user-attachments/assets/81e23c2c-a860-4d9e-b8f4-e6bce2a80a83)

Reply:
Yes, this way we get rid of the old code immediately. This is also recommended in [R Packages](https://r-pkgs.org/lifecycle.html#deprecating-a-function).